gamestonk_terminal/config_neural_network_models.py

```diff
@@ -23,19 +23,36 @@
     {"Dense": {"units": 100, "activation": "relu"}},
     {"Dense": {"units": 80, "activation": "relu"}},
     {"Dense": {"units": 30, "activation": "relu"}},
-    {"Dense": {"activation": "linear"}},
+    {"Dense": {"activation": "relu"}},
 ]
 
 Recurrent_Neural_Network = [
     {"SimpleRNN": {"units": 100, "activation": "linear", "return_sequences": True}},
     {"SimpleRNN": {"units": 50, "activation": "linear", "return_sequences": True}},
     {"Dropout": {"rate": 0.2}},
     {"SimpleRNN": {"units": 21, "activation": "linear", "return_sequences": False}},
-    {"Dense": {"activation": "linear"}},
+    {"Dense": {"activation": "relu"}},
 ]
 
 Long_Short_Term_Memory = [
     {"LSTM": {"units": 25, "activation": "tanh", "return_sequences": True}},
     {"LSTM": {"units": 15, "activation": "tanh", "return_sequences": False}},
-    {"Dense": {"activation": "linear"}},
+    {"Dense": {"activation": "relu"}},
 ]
+
+Convolutional = [
+    {"Conv1D": {"filters": 20, "kernel_size": 10, "activation": "relu"}},
+    {"MaxPool1D": {"pool_size": 2}},
+    {"Conv1D": {"filters": 10, "kernel_size": 5, "activation": "relu"}},
+    {"MaxPool1D": {"pool_size": 2}},
+    {"Flatten": {}},
+    {"Dense": {"activation": "relu"}},
+]
+
+# Optimizers :  https://www.tensorflow.org/api_docs/python/tf/keras/optimizers
+# Adam, Adagrad, Adadelta, Adamax,Ftrl, Nadam, Rmsprop, Sgd
+Optimizer = "Adam"
```

Comment:
What do you think about also having:
* Learning Rate
* Loss Function
* Pre-processing technique
* Training Epochs

And we could leave for command line just: Input days, Prediction days

Reply:
So I think LR and Epochs should be command line (so you don't have to restart every time if you wanna change one of them).  Loss and preprocess is probably good!